gateway/api_loader.go

```diff
@@ -890,7 +890,14 @@ func (gw *Gateway) loadApps(specs []*APISpec) {
 		track404Logs: gwConf.Track404Logs,
 	}
 
-	router := gw.loadControlAPIEndpoints()
+	db, err := NewDatabase("/opt/tyk-gateway/mongodb.yml")
+	if err != nil {
+		log.Fatalf("cant createa db, err: %s", err)
+	}
```

Comment:
1. Hardcode doesn't seem right.
2. pls don't use fatal here

Reply:
`Hardcode doesn't seem right.` -  what do you mean?
`pls don't use fatal here` - fixed

Reply:
I mean it's good to have a constant or a method to get the path to the volume rather than writing it by hand.

Reply:
fixed